doc/source/ray-observability/state/state-api.rst

```diff
@@ -0,0 +1,303 @@
+
+    ray get tasks <TASK_ID>
+
+.. code-block:: text
+
+    ---
+    func_or_class_name: task_running_300_seconds
+    language: PYTHON
+    name: task_running_300_seconds()
+    required_resources:
+        CPU: 1.0
+    runtime_env_info:
+        runtime_env_config:
+            eager_install: true
+            setup_timeout_seconds: 600
+        serialized_runtime_env: '{}'
+    scheduling_state: RUNNING
+    task_id: 32d950ec0ccf9d2affffffffffffffffffffffff01000000
+    type: NORMAL_TASK
+
+You can also access logs through ``ray logs`` API.
+
+.. code-block:: bash
+
+    ray list actors
+    ray logs --actor-id <ACTOR_ID>
+
+.. code-block:: text
+
+    --- Log has been truncated to last 1000 lines. Use `--tail` flag to toggle. ---
+
+    :actor_name:Actor
+    Actor created
+
+
+Key Concepts
+------------
+Ray state APIs allow you to access **states** of **resources** through **summary**, **list**, and **get** APIs.
+
+- **states**: The state of the cluster of corresponding resources. States consist of immutable metadata (e.g., actor's name) and mutable states (e.g., actor's scheduling state or pid).
+- **resources**: Resources created by Ray. E.g., actors, tasks, objects, placement groups, and etc. 
+- **summary**: API to return the summarized view of resources.
+- **list**: API to return every individual entity of resources.
+- **get**: API to return a single entity of resources in detail.
+
+Summary 
+-------
+Return the summarized information of the given Ray resource (objects, actors, tasks).
+It is recommended to start monitoring states through summary APIs first. When you find anomalies
+(e.g., actors running for a long time, tasks that are not scheduled for a long time),
+you can use ``list`` or ``get`` APIs to get more details for an individual abnormal resource.
+
+E.g. Summarize all actors (e.g. number of alive actors, different actor classes, etc)
+
+.. code-block:: bash
+
+    ray summary actors
+
+E.g. Summarize all tasks (e.g. task count in different states, type of different tasks, etc)  
+
+.. code-block:: bash
+
+    ray summary tasks
+
+E.g. Summarize all objects (e.g. total number of objects, size of all objects, etc) 
+
+.. code-block:: bash
+
+    # To get callsite info, set env variable `RAY_record_ref_creation_sites=1` when starting the ray cluster
+    # RAY_record_ref_creation_sites=1 ray start --head
+    ray summary objects 
+
+List
+----
+
+Get a list of resources, possible resources include: 
+
+:ref:`Actors <actor-guide>`
+:ref:`Tasks <ray-remote-functions>`
+:ref:`Objects <objects-in-ray>`
+:ref:`Jobs <jobs-overview>`
+:ref:`Placement Groups <ray-placement-group-doc-ref>`
+:ref:`Nodes (Ray worker nodes)`
+:ref:`Workers (Ray worker processes)`
+:ref:`Runtime environments <runtime-environments>`
+
+E.g. List all nodes
+
+.. code-block:: bash
+
+    ray list nodes 
+
+E.g. List all placement groups
+
+.. code-block:: bash
+
+    ray list placement-groups
+
+Listing resources with one or multiple filters 
```

Comment:
This is not rendered well. 
<img width="922" alt="image" src="https://user-images.githubusercontent.com/11676094/180499847-f7d82521-bc95-49da-a339-a6c7a7f1625b.png">